### PR TITLE
Refactor widgets

### PR DESCRIPTION
**Pull request by MrPetovan**
- Move widget templates to subfolder
- Add new generic filter widget templates to replace category, network and filer filters
- Update styles

Preparatory work for the upcoming contact relationship filter.

### CODE REVIEW

File: src/Module/Contact.php [hunk 1]
```diff
@@ -664,30 +654,24 @@ public static function content($update = 0)
...
 		}
 
-		$blocked = false;
-		$hidden = false;
-		$ignored = false;
-		$archived = false;
-		$all = false;
-
 		// @TODO: Replace with parameter from router
-		if (($a->argc == 2) && ($a->argv[1] === 'all')) {
```
Comment (nupplaphil) [inline src/Module/Contact.php]: Just a question: So `contact/all` isn't a valid path anymore?

**Comment by nupplaphil** [comment]
Puh, I'm not very familiar with the smarty-templates and CSS. For me it looks fine, but maybe somebody else can take a short look on the CSS stuff

File: src/Module/Contact.php (hunk 1 above)
Comment (MrPetovan) [reply src/Module/Contact.php]: No and I see where this is going, patch incoming.

**Comment by nupplaphil** [comment]
I'd be fine with the PR, @mrpetovan are you blocked because of this PR?

**Comment by MrPetovan** [comment]
#7156 is blocked by it, yes.

**Comment by AlfredSK** [comment]
...in src/Module/Contact.php line 263.

**Pull request merged**